### ISSUE TITLE
An error occurred while I was running the software

### ISSUE THREAD

**Issue by jingydz** [esteinig/netviewP#1]
Hi,
May I ask you a question?When I installed the software, there was an error when I tried to run it.
[root@GenEngine example]# python2.7 ../netview.py -f oyster.map -a oyster.csv -p --quality --distance asd --algorithm brute --mst-off
()
[01:16:09]      ---------------------------------
[01:16:09]               NETVIEW P v.0.7.1
[01:16:09]      ---------------------------------
('[01:16:09]\tFile =', 'OYSTER.MAP')
Traceback (most recent call last):
  File "../netview.py", line 882, in <module>
    main()
  File "../netview.py", line 46, in main
    dat.readData(commands.arg_dict['data_file'], f='plink', sep=commands.arg_dict['sep'])
  File "../netview.py", line 437, in readData
    _read_plink(infile, file, sep)
  File "../netview.py", line 380, in _read_plink
    self.meta_data["sex"] = [i[4] for i in meta]
IndexError: list index out of range
I'm using python 2.7.
What should I do?
Look forward to your reply!

**Comment by esteinig** [comment]
Hi @jingydz can you run it with Python 3.4+ and see what happens?

**Comment by jingydz** [comment]
Thank you very much for your answer!
I changed the python version to 3.4 and it worked.
Best,
Zhang.

**Comment by jingydz** [comment]
😂😂😂
Hi,
Sorry I ran my file with this error, I tried to modify the netview.py file, but failed.
(python34) [zhangjingjing@GenEngine 0.7.1]$ ./netview.py -f wqspalax.ped -a wqspalax.csv -p --quality --distance asd --algorithm brute --mst-off

[21:01:45]      ---------------------------------
[21:01:45]               NETVIEW P v.0.7.1         
[21:01:45]      ---------------------------------
[21:01:45]      File = WQSPALAX.PED
Traceback (most recent call last):
  File "./netview.py", line 882, in <module>
    main()
  File "./netview.py", line 46, in main
    dat.readData(commands.arg_dict['data_file'], f='plink', sep=commands.arg_dict['sep'])
  File "./netview.py", line 437, in readData
    _read_plink(infile, file, sep)
  File "./netview.py", line 368, in _read_plink
    ids.append(content[1])
IndexError: list index out of range
And I tried to compare the example files and found that the map files were very different.
(python34) [zhangjingjing@GenEngine example]$ less -SN oyster.map 
      1 0       snp1    0       10001
      2 0       snp2    0       10002
      3 0       snp3    0       10003
      4 0       snp4    0       10004
(python34) [zhangjingjing@GenEngine 0.7.1]$ less -SN wqspalax.map 
      1 1       1:2568  0       2568
      2 1       1:2569  0       2569
      3 1       1:2570  0       2570
      4 1       1:2571  0       2571
      5 1       1:2572  0       2572
      6 1       1:2573  0       2573
      7 1       1:2574  0       2574
      8 1       1:2575  0       2575
      9 1       1:2576  0       2576
     10 1       1:2577  0       2577
The first is the example of the map file, the second is my map file, I would like to ask whether the problem is here?
Look forward to your reply!
